nixos/modules/services/networking/onedrive.nix

```diff
@@ -0,0 +1,88 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.onedrive;
+
+  onedriveLauncher =  pkgs.writeShellScriptBin
```

Comment:
I personally don't like the use of a launcher like this as you cannot individually start and stop the onedrive instances as the launcher will process all you have configured. IMHO, it's far better to just create the `onedrive@` unit and have the user activate what needs to be activated. If multiple instances are expected, you could wrap those in a `onedrive.target` and have `default.target` pull in `onedrive.target`. That gives you a way to start them all at one go as well as start/stop them individually.

Reply:
This launcher only helps in one way- starting all (say 4) onedrive service instances. This launcher is auto-started because it is `wantedBy` `default.target`. One can manually start the launcher. 

One can also manually start and stop individual onedrive client instance services.